.github/workflows/test.js.yml

```diff
@@ -1,11 +1,13 @@
 # This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node
 # For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
 
-name:  CI
+name:  Unit Tests
 
 on:
   pull_request:
     branches: [ main ]
+  push: 
```

Comment:
nice, there should be branch protections on main but maybe I configured em wrong cause I definitely accidentally pushed a commit to main

Reply:
Yeah 👍🏻 There should be lol, but also, this will then run the tests after we merge to main (I think?)